diff --git a/src/components/ImageGallery.jsx b/src/components/ImageGallery.jsx
index bfe79b72fe629fc6a4028b909d0c7208e8f80ab3..a65fe007bdc80caa728935d5d41ce0890bc1170c 100644
--- a/src/components/ImageGallery.jsx
+++ b/src/components/ImageGallery.jsx
@@ -5,13 +5,15 @@ import debounce from "lodash-es/debounce";
 import isEqual from "react-fast-compare";
 import ResizeObserver from "resize-observer-polyfill";
 import { LEFT, RIGHT, UP, DOWN } from "react-swipeable";
-import { arrayOf, bool, func, number, oneOf, shape, string } from "prop-types";
-import Item from "src/components/Item";
-import Fullscreen from "src/components/controls/Fullscreen";
-import LeftNav from "src/components/controls/LeftNav";
-import RightNav from "src/components/controls/RightNav";
-import PlayPause from "src/components/controls/PlayPause";
-import SwipeWrapper from "src/components/SwipeWrapper";
+import Item from "./Item";
+import Fullscreen from "./controls/Fullscreen";
+import LeftNav from "./controls/LeftNav";
+import RightNav from "./controls/RightNav";
+import PlayPause from "./controls/PlayPause";
+import SwipeWrapper from "./SwipeWrapper";
+import pkg from 'prop-types';
+
+const {arrayOf, bool, func, number, oneOf, shape, string} = pkg;
 
 const screenChangeEvents = [
   "fullscreenchange",
diff --git a/src/components/Item.jsx b/src/components/Item.jsx
index f4e18d1221bdef6074247490f7b3007694653ca1..5434126e657a32222ca53c4ea8a3021dd80108ef 100644
--- a/src/components/Item.jsx
+++ b/src/components/Item.jsx
@@ -1,5 +1,6 @@
 import React from "react";
-import { bool, func, string } from "prop-types";
+import pkg from 'prop-types';
+const { bool, func, string } = pkg;
 
 const defaultProps = {
   description: "",
diff --git a/src/components/SVG.jsx b/src/components/SVG.jsx
index 58aa9bcb04ecceb8709ebfa7e5ab5fdf666c49bd..6c3445365faf105ad1d646fc7b2239f776f19363 100644
--- a/src/components/SVG.jsx
+++ b/src/components/SVG.jsx
@@ -1,5 +1,6 @@
 import React from "react";
-import { number, oneOf, string } from "prop-types";
+import pkg from 'prop-types';
+const { number, oneOf, string } = pkg;
 
 const left = <polyline points="15 18 9 12 15 6" />;
 const right = <polyline points="9 18 15 12 9 6" />;
diff --git a/src/components/SwipeWrapper.jsx b/src/components/SwipeWrapper.jsx
index f0c50cec52b44784a2a7a8da8de611c87ec769a1..48494e19aebffc45d196d00dbc6781d8cdf08496 100644
--- a/src/components/SwipeWrapper.jsx
+++ b/src/components/SwipeWrapper.jsx
@@ -1,6 +1,7 @@
 import React from "react";
-import { string, node, number, func } from "prop-types";
 import { useSwipeable } from "react-swipeable";
+import pkg from 'prop-types';
+const { string, node, number, func } = pkg;
 
 const defaultProps = {
   className: "",
diff --git a/src/components/controls/Fullscreen.jsx b/src/components/controls/Fullscreen.jsx
index 753469cac8e3c4308d1c8540a719cf96cb335994..9ac82412a0e5844aa4f26684689b4e62007a7238 100644
--- a/src/components/controls/Fullscreen.jsx
+++ b/src/components/controls/Fullscreen.jsx
@@ -1,6 +1,7 @@
 import React from "react";
-import { bool, func } from "prop-types";
-import SVG from "src/components/SVG";
+import SVG from "../SVG";
+import pkg from 'prop-types';
+const { bool, func } = pkg;
 
 const Fullscreen = React.memo(({ isFullscreen, onClick }) => {
   return (
diff --git a/src/components/controls/LeftNav.jsx b/src/components/controls/LeftNav.jsx
index 88a64e63b039324812df9d278ece8507e204a2f1..bb42235a82cdea4745707f8bd786264d9e9235ff 100644
--- a/src/components/controls/LeftNav.jsx
+++ b/src/components/controls/LeftNav.jsx
@@ -1,6 +1,7 @@
 import React from "react";
-import { bool, func } from "prop-types";
-import SVG from "src/components/SVG";
+import SVG from "../SVG";
+import pkg from "prop-types";
+const { bool, func } = pkg;
 
 const LeftNav = React.memo(({ disabled, onClick }) => {
   return (
diff --git a/src/components/controls/PlayPause.jsx b/src/components/controls/PlayPause.jsx
index 80b7881ee52480cfb6902d0ad409c5d4f2544d69..32933f24233c5bfd8f55df3bc16a0ef68d8b32d8 100644
--- a/src/components/controls/PlayPause.jsx
+++ b/src/components/controls/PlayPause.jsx
@@ -1,6 +1,7 @@
 import React from "react";
-import { bool, func } from "prop-types";
-import SVG from "src/components/SVG";
+import SVG from "../SVG";
+import pkg from "prop-types";
+const { bool, func } = pkg;
 
 const PlayPause = React.memo(({ isPlaying, onClick }) => {
   return (
diff --git a/src/components/controls/RightNav.jsx b/src/components/controls/RightNav.jsx
index 54280e3d6697dc6194419f5a278e0c4f01c243cf..3029061e0374f9739d6c4337950fdd53bc2e3bb1 100644
--- a/src/components/controls/RightNav.jsx
+++ b/src/components/controls/RightNav.jsx
@@ -1,6 +1,7 @@
 import React from "react";
-import { bool, func } from "prop-types";
-import SVG from "src/components/SVG";
+import SVG from "../SVG";
+import pkg from "prop-types";
+const { bool, func } = pkg;
 
 const RightNav = React.memo(({ disabled, onClick }) => {
   return (
